package/aur/PKGBUILD

```diff
@@ -11,20 +12,49 @@ license=('MIT' 'Apache')
 depends=('gcc-libs')
 # Version required for `strip`: '1.45.0-nightly' (https://github.com/rust-lang/rust/issues/72110)
 makedepends=(cargo)
-source=($pkgname-${pkgver//_/-}.tar.gz::$url/archive/v${pkgver//_/-}.tar.gz)
+# shellcheck allow=SC2140
+source=("${pkgname}-${pkgver//_/-}.tar.gz::${url}/archive/v${pkgver//_/-}.tar.gz")
 sha512sums=('0dc069a118ff1fa87efce75e7ecc35c8a047721146c23d2830ead72fb963f9f4d1ff81973074b0d64a615f5b53dc7a84c4714fda6e4587582f26905ec23c8c1a')
```

Comment:
does it make sense to hardcode the SHA?

Reply:
Yes! The build script is updated on each release and pushed to [AUR](https://aur.archlinux.org/) manually.
(Verions, Hash have to be adjusted each time anyway)